_posts/2020-08-26-enforce-custom-resource-policies-with-opa-gatekeeper.md

```diff
@@ -0,0 +1,435 @@
+---
+layout: post
+title:  "Enforce Custom Resource policies with Open Policy Agent Gatekeeper"
+date: 2020-08-26
+author: jakub_scholz
+---
+
+In our [last blog post](https://strimzi.io/blog/2020/08/05/using-open-policy-agent-with-strimzi-and-apache-kafka/) about [Open Policy Agent (OPA)](https://www.openpolicyagent.org/), we looked at how it can be used with Strimzi for Kafka authorization.
```

Comment:
```suggestion
In our [last blog post](https://strimzi.io/blog/2020/08/05/using-open-policy-agent-with-strimzi-and-apache-kafka/) we talked about [Open Policy Agent (OPA)](https://www.openpolicyagent.org/), and how it can be used with Strimzi for Kafka authorization.
```